aws-spi-pekko-http/src/test/resources/logback-test.xml

```diff
@@ -15,7 +15,7 @@
         <appender-ref ref="STDOUT" />
     </root>
 
-    <logger name="io.netty" level="trace" additivity="false">
+    <logger name="io.netty" level="info" additivity="false">
```

Comment:
The `SdkAsyncHttpClientH1TestSuite` uses netty, and netty bootstrapping is very verbose, so I increased the log level